docs/reference/commandline/dockerd.md

```diff
@@ -1279,3 +1282,25 @@ $ sudo dockerd \
         --data-root=/var/lib/docker-bootstrap \
         --exec-root=/var/run/docker-bootstrap
 ```
+
+### Default network options
+
+The `--default-network-opt` flag, and the equivalent `default-network-opts` key
+in the configuratio file, lets you specify custom default options for custom
```

Comment:
"custom" twice in this sentence reads a bit awkward. Perhaps drop the first one;

```suggestion
in the configuratio file, lets you specify default options for custom
```

Reply:
also a typo :D